Sources/Build/Sanitizers.swift

```diff
@@ -0,0 +1,100 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2018 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+import Basic
+import Utility
+
+/// A set of enabled runtime sanitizers.
+public struct EnabledSanitizers {
+    public enum Sanitizer: String {
+        case address
+        case thread
+    }
+
+    /// A set of enabled sanitizers.
+    private var sanitizers: Set<Sanitizer> = []
+
+    public init() {}
+    public init(from list: [Sanitizer]) {
```

Comment:
We can make this `public init(from list: [Sanitizer] = [])` and avoid the above `init()`.